facefusion/processors/modules/face_swapper.py

```diff
@@ -544,7 +546,7 @@ def forward_swap_face(source_face : Face, crop_vision_frame : VisionFrame) -> Vi
 			if model_type in [ 'blendswap', 'uniface' ]:
 				face_swapper_inputs[face_swapper_input.name] = prepare_source_frame(source_face)
 			else:
-				face_swapper_inputs[face_swapper_input.name] = prepare_source_embedding(source_face)
+				face_swapper_inputs[face_swapper_input.name] = prepare_source_embedding(source_face, target_face)
```

Comment:
do not mix embedding preparation and embedding mixing - it belongs to it's own method